llvm/docs/Passes.rst

```diff
@@ -193,14 +196,15 @@ memory operations it depends on.  It builds on alias analysis information, and
 tries to provide a lazy, caching interface to a common kind of alias
 information query.
 
-``module-debuginfo``: Decodes module-level debug info
------------------------------------------------------
+``print<module-debuginfo>``: Decodes module-level debug info
+------------------------------------------------------------
 
 This pass decodes the debug info metadata in a module and prints in a
 (sufficiently-prepared-) human-readable form.
 
-For example, run this pass from ``opt`` along with the ``-analyze`` option, and
-it'll print to standard output.
+For example, run this pass from ``opt`` along with the
```

Comment:
`along with the ... option` is confusing since that's true for every pass listed here

Reply:
Ok, I agree. This was only here to refer to ``-analyze`` in the past. I'll drop this example completely as there is nothing special with this pass compared to all other passes listed in the document.

As usuel @aeubanks, thanks a lot for prompt feedback and good advice!